internal/command/setup_step1.go

```diff
@@ -134,7 +135,10 @@ func (c *Commands) SetupStep1(ctx context.Context, step1 *Step1) error {
 					EmailAddress:    organisation.Owner.Email,
 					IsEmailVerified: true,
 				},
-			}, orgIAMPolicy, pwPolicy, nil, false)
+			}, orgIAMPolicy, pwPolicy,
+			nil, //TODO: Code Generator missing!
+			nil, //TODO: Code Generator missing!
```

Comment:
just as notice: for v2 we'll have to set them up in step1 / createIAM / createTenant and pass them along